cmd/utils/utils.go

```diff
@@ -57,3 +57,14 @@ func GoFmt(appDir string) error {
 
 	return nil
 }
+
+// GoModTidy runs "go mod tidy" in a selected directory
+func GoModTidy(appDir string) error {
+	if err := ExecuteCmd("go",
+		[]string{"mod", "tidy"},
+		appDir); err != nil {
+		return err
+	}
+
```

Comment:
```suggestion
	err := ExecuteCmd("go",
		[]string{"mod", "tidy"},
		appDir);

	if err != nil {
		return err
	}
```

nit change. should probably also just run go fmt

Reply:
Im following the format used in previous cmds, maybe would be good to stick with same format until have some standards (?)